ts/screens/authentication/idpAuthSessionHandler.tsx

```diff
@@ -309,7 +314,8 @@ export const AuthSessionPage = () => {
                           result,
                           getEitherLoginResult,
                           E.fold(
-                            e => handleLoginFailure(e.errorCode),
+                            e =>
+                              handleLoginFailure(e.errorCode || e.errorMessage),
```

Comment:
In this way the `handleLoginFailure` first parameter will be either `errorCode` or `errorMessage`, doing so the second parameter (meant for handling the error messages) is never used. What do you think about removing the second parameter from `handleLoginFailure`?

Reply:
I've seen that `handleLoginFailure` needs to handle `code` and `message` in different ways, so in this case the solution could be calling `handleLoginFailure` with both `e.errorCode` and `e.errorMessage` respectively as first and second argument.

Reply:
yes good catch